taxi/forms.py

```diff
@@ -0,0 +1,56 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+
+
+class LicenseNumberBaseForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data.get("license_number")
+        if len(license_number) != 8:
+            raise forms.ValidationError(
+                "License number must be 8 characters long"
+            )
+        if (
+                not license_number[:3].isupper()
+                or not license_number[:3].isalpha()
+        ):
+            raise forms.ValidationError(
+                "First 3 characters must be uppercase"
+            )
+        if not license_number[3:].isdigit():
```

Comment:
incorrect number of last characters that you should check

Reply:
But it checks correctly. There should be numbers from 4 characters, and to make sure there are no more, I check above
`        if len(license_number) != 8:
            raise forms.ValidationError(
                "License number must be 8 characters long"
            )`